backend/substrapp/views/compositetraintuple.py

```diff
@@ -110,8 +104,6 @@ def retrieve(self, request, *args, **kwargs):
         try:
             data = self._retrieve(pk)
         except LedgerError as e:
-            return Response({'message': str(e.msg)}, status=e.status)
-        except Exception as e:
-            return Response({'message': str(e)}, status.HTTP_400_BAD_REQUEST)
-        else:
-            return Response(data, status=status.HTTP_200_OK)
+            raise exceptions.from_ledger_error(e)
+
+        return Response(data, status=status.HTTP_200_OK)
```

Comment:
This is the only use of `from_ledger_error `, everywhere else you kept the `return Response({'message': str(e.msg)}, status=e.status)`.
I think we should either use from_ledger_error everywhere or nowhere. In this specific case, I'd be for using it nowhere and then in a separate PR converting all manual returns when catching an exception with raising excepptions.from_ledger_error.

Reply:
100% agree, my last commit was not intended to be part of this PR (it's named wip...).
I will move it to a distinct PR.

Reply:
Done